sopel/bot.py

```diff
@@ -1030,12 +1034,11 @@ def error(
         self,
         trigger: Optional[Trigger] = None,
         exception: Optional[BaseException] = None,
-    ):
+    ) -> None:
         """Called internally when a plugin causes an error.
 
         :param trigger: the ``Trigger``\\ing line (if available)
```

Comment:
Nit: could probably drop the reference to the type here? I would probably also omit the word "line"

Reply:
Maybe this would work, but I wouldn't want to take too much context out of the docstring.

```suggestion
        :param trigger: the ``Trigger`` that led to the error, if available
```

Reply:
Or maybe... ?

```suggestion
        :param trigger: the line leading to that error (if available)
```

Reply:
Another idea:

```suggestion
        :param trigger: the IRC line that caused the error (if available)
```

This feels a lot like something I always have to spend a lot of time on at work 😂